src/Lumper.UI/ViewModels/Shared/Pakfile/PakfileEntryViewModel.cs

```diff
@@ -34,6 +28,8 @@ protected MemoryStream DataStream
 
     public long? CompressedSize => BaseEntry.CompressedSize;
 
+    [ObservableAsProperty]
+    public string? Hash { get; }
     protected PakfileEntryViewModel(PakfileEntry baseEntry, BspNode parent)
```

Comment:
Tiny formatting blip due to commit ordering, isn't in final changes. Not worth fixing!